data/data.json

```diff
@@ -8653,6 +8653,49 @@
    "aciGbpServerContainer": "noiro/gbp-server:5.1.1.0.1ae238a",
    "aciOpflexServerContainer": "noiro/opflex-server:5.1.1.0.1ae238a"
   },
+  "v1.20.15-rancher1-1": {
+   "etcd": "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
+   "alpine": "rancher/rke-tools:v0.1.78",
```

Comment:
we need to bump rke-tools to v0.1.79 for all versions

Reply:
@doflamingo721 could you please also update them here https://github.com/rancher/kontainer-driver-metadata/pull/780 so both PRs would be ready to merge